src/Microsoft.AspNet.Mvc.TagHelpers/FormTagHelper.cs

```diff
@@ -98,7 +87,7 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
                                                         Action,
                                                         Controller,
                                                         routeValues,
-                                                        Method,
+                                                        method: null,
```

Comment:
don't need to pass user's `method` to the `IHtmlGenerator`; final result is the same as before because `TagBuilder` attributes do not overwrite anything already in the `TagHelperOutput`.